### PR TITLE
TRIDENT-18109 - Add note on extracted fields

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a note to this article:
https://help.sumologic.com/docs/cse/rules/write-match-rule/

It also adds text to "with a summary" fields in other articles, because it bugged me that we had no text there. 

Issue number: 
[TRIDENT-18109](https://sumologic.atlassian.net/browse/TRIDENT-18109)

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
We'll hold off publishing this one until [TRIDENT-18108](https://jasklabs.atlassian.net/browse/TRIDENT-18108) is closed. Customers can't use the example in the note until that ticket is resolved.

File: docs/cse/rules/write-aggregation-rule.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ On the right side of the Rules Editor, in the **Then Create a Signal** section,
 <img src={useBaseUrl('img/cse/then-create-a-signal.png')} alt="then-create-a-signal.png" width="400"/>
 
 1. **On Entity**. Use the pull-down list to select one or more Entity fields, for example, an IP address, MAC address, hostname, and so on. When the rule is triggered, it will fire a Signal on each of the Entity fields you select.  
-1. **with the summary**. 
+1. **with the summary**.  Enter a brief summary describing what causes the Rule to create a Signal.
```
Comment (kimsauce) [inline docs/cse/rules/write-aggregation-rule.md:98]:
```suggestion
1. **with the summary**. Enter a brief summary describing what causes the Rule to create a Signal.
```

File: docs/cse/rules/write-chain-rule.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ If you use the Test Rule feature on a rule that has one or more [Rule Tuning Exp
 ## Configure “Then Create a Signal” settings
 
 1. **On Entity**. Define the Entity field — for example, an IP address, hostname, and so on — in the Record that the resulting Signal should be associated with. (In CSE, an Insight is a set of Signals with the same Entity field.) Select a value from the pull-down list. 
-1. **with the summary.**
+1. **with the summary.**  Enter a brief summary describing what causes the Rule to create a Signal.
```
Comment (kimsauce) [inline docs/cse/rules/write-chain-rule.md:66]:
```suggestion
1. **with the summary.** Enter a brief summary describing what causes the Rule to create a Signal.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fixes

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce.

**Comment by jpipkin1** [comment]
@kimsauce, I made your suggested changes. Could you give approval to this one so it's ready to go whenever Dev gets [APP-2464](https://jasklabs.atlassian.net/browse/APP-2464) resolved?

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**